homeassistant/components/nest/legacy/__init__.py

```diff
@@ -96,7 +96,7 @@ def nest_update_event_broker(hass, nest):
     _LOGGER.debug("Stop listening for nest.update_event")
 
 
-async def async_setup_legacy(hass, config):
+async def async_setup_legacy(hass, config) -> bool:
```

Comment:
Missing parameter typing. Please type the whole signature when adding type annotations.

Reply:
Will do.